app/registries/foreman/settings/general.rb

```diff
@@ -63,7 +63,7 @@
       full_name: N_('Append domain names to the host'))
     setting('outofsync_interval',
       type: :integer,
-      description: N_('Duration in minutes after servers are classed as out of sync. You can override this on hosts by adding a parameter "outofsync_interval".'),
+      description: N_('Duration in minutes after servers are classed as out of sync. This setting is overridden by specific settings from content management tools (e.g. puppet_inteval, ansible_interval).'),
```

Comment:
```suggestion
      description: N_('Duration in minutes after servers are classed as out of sync. This setting is overridden by specific settings from config management tools (e.g. puppet_inteval, ansible_interval).'),
```
You can split this into multiple lines by:
```suggestion
      description: N_('Duration in minutes after servers are classed as out of sync. '\
                               'This setting is overridden by specific settings from content management tools (e.g. puppet_inteval, ansible_interval).'),
```